olp-cpp-sdk-core/include/olp/core/http/Network.h

```diff
@@ -90,7 +105,24 @@ class CORE_API Network {
    *
    * @param headers The default headers.
    */
-  virtual void SetDefaultHeaders(Headers) {}
+  virtual void SetDefaultHeaders(Headers headers);
+
+  /**
+   * @brief Set the current statistics bucket.
+   *
+   * @param[in] bucked_id The bucket ID.
+   */
+  virtual bool SetCurrentBucket(uint8_t bucket_id);
```

Comment:
Do we need to return anything? Can this operation fail?
If yes, then please add doxygen explaining when do we return true and when do we return false.